manifest.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should we move this file, `index.html` and `images/icon.png` into `public/` (and removing it from `.gitignore`)?